### ISSUE TITLE
Редактор слэшит кавычки на любом парсере, использовать HTML невозможно

### ISSUE THREAD

**Issue by murvanidze** [maxsite/cms#267]
Добрый день! 
Проблема появилась ещё на зимних версиях и не ушла в версии 100. 
Независимо от обработчика HTML, движок слэшит кавычки в HTML-тегах и превращает их в жмых. 
С парсером **Default** в админке в **предпросмотре** теги выходят **нормально**, но на морде - всё равно жмых. 
Отключал ббкоды, парсеры, включал, всё перепробовал - всё равно жмых. 
**На старых версиях (до осени 2017) на том же хосте всё нормально** с любым парсером. 

Прошу помочь, невозможно пользоваться системой! 
Приходится подпиливать кавычки в базе, но это помогает только до следующего редактирования записи. 
На Максайте не первый год, очень комфортно, надеюсь проблема решаемая. 
Спасибо!

**Скриншоты:** 
Парсер симпле (морда)

![sshot-2018-11-04- 06-04-40 -1](https://user-images.githubusercontent.com/44739518/47959877-8cfb5d80-dff7-11e8-94e4-be4a261f2e99.png)

Парсер симпле (код)

![sshot-2018-11-04- 06-10-30 -1](https://user-images.githubusercontent.com/44739518/47959913-5eca4d80-dff8-11e8-83ee-2431bc5526c0.png)

Парсер дефолт (морда)

![sshot-2018-11-04- 06-12-02 -1](https://user-images.githubusercontent.com/44739518/47959925-b1a40500-dff8-11e8-887b-449f96ac549c.png)

Парсер дефолт (предпросмотр работает норм!)

![sshot-2018-11-04- 06-12-29 -1](https://user-images.githubusercontent.com/44739518/47959930-cb454c80-dff8-11e8-9e38-cc13246a7fd4.png)

**Страница:** 
http://hellens.mda.link/page/summer-mix-2018-catalog

Сервер: 

![sshot-2018-11-04- 06-15-06 -1](https://user-images.githubusercontent.com/44739518/47959938-1c554080-dff9-11e8-9781-c87c698fcea1.png)

**Comment by maxsite** [comment]
Редактор стандартный? 

Попробуйте другим браузером.

**Comment by maxsite** [comment]
Да, и приведите полный текст, на котором возникает пробема, чтобы можно было проверить у себя.

**Comment by murvanidze** [comment]
Привет! 
Спасибо, что вы здесь! 
Попробовал другие браузеры - не в этом соль, пходу. 
И проблема точно не в тексте. 

Прикопал ещё и выяснил - проблема возникает при фоновом сохранении. То есть: 
Создаём страницу - нажимаем Сохранить - всё норм, кавычки норм. 
Редактируем страницу - жмём Сохранить - всё норм, кавычки норм. 
А если нажмём "Сохранить в фоне" - получаем жмых. 

Проблема возникает именно в момент переноса текста из редактора в базу посредством "Сохранить в фоне". 

Текст не имеет значения, попробовал и однострочный код спан. 
Могу дать доступ, если надо.

**Comment by maxsite** [comment]
Фоновое и обычное сохранение — это одна функция. Приведите всё-таки текст. По симптомам это похоже на экранирование. Но неясно — это уровень браузера, или post-запрос в PHP. Если PHP, то экранирование нужно настроить на сервере. Если браузер, то смотрите в другом браузере, если повторяется, то попробуйте на другом компе (но это совсем сложный случай).

**Comment by murvanidze** [comment]
текст такой: 
`<span class="t-red">lalala</span>` 

потестил в хроме. то же самое. 
на более ранних версиях на этом же сервере этой проблемы нет и сейчас, в любом браузере.

**Comment by maxsite** [comment]
А на выходе (страница сайта) этот текст что выдает?

**Comment by murvanidze** [comment]
Если сохранить по классике, с перезагруской едит_паге: 
`<span class="t-red">lalala</span>`
Если **сохранить в фоне**,то так: 
`<span class=\"t-red\">lalala</span>`
Если открыть её потом заново в редакторе и ещё раз сохранить в фоне, то так: 
`<span class=\\\"t-red\\\">lalala</span>`

Проблема однозначно проявляется ТОЛЬКО при использовании фонового сохранения. Может там функция и одна и та же, но через фоновое текст попадает в функцию уже отслэшенный? 

Браузер у меня Фокс 63 х64, проверял также в хроме 56. 

Поскольку найден путь обхода, проблема уже не критична, но она однозначно есть.

**Comment by maxsite** [comment]
Это точно экранирование. Значит дело вот в чем. Обработка данных проходит через функцию parse_str(), на которую влияют настройки экранирования php: magic_quotes_XX. В версии php 5.4 всё это дело объявлено устаревшим, а на php 5.3 зависит от настроек сервера (файл php.ini).  То есть по идее вам будет достаточно либо просто перейти на новую версию php, либо поиграть настройками php.ini http://php.net/manual/ru/security.magicquotes.disabling.php
